.github/workflows/check_make_vtadmin_authz_testgen.yml

```diff
@@ -6,10 +6,22 @@ jobs:
     name: Check Make vtadmin_authz_testgen
     runs-on: ubuntu-latest
     steps:
+    - name: Check if workflow needs to be skipped
+      id: skip-workflow
+      run: |
+        skip=0
+        if [[ "${{github.event.pull_request}}" ==  "" ]] && [[ "${{github.ref}}" != "refs/heads/main" ]] && [[ ! "${{github.ref}}" =~ "refs/heads/release-.*\.0" ]] && [[ ! "${{github.ref}}" =~ "refs/tags/.*" ]]; then
```

Comment:
Nit, but I think for production code we should try be more precise with regex's. For example:
```
"^refs/heads/release-[0-9]+\.[0-9]$"
```

The second part is so that we don't have to find places like this and fix them if we ever decide to start doing minor releases (e.g. 14.1.0).

Reply:
Done!